content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Mengatasi error validasi
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Pada bagian ini, kita akan membahas dan menyelesaikan error validasi AMP dari halaman AMP.  Perhatikan bahwa error mungkin muncul dalam urutan yang berbeda di konsol Anda.
+
+## Sertakan charset
+
+Kita akan memulai dengan memperbaiki error berikut:
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.